### PR TITLE
[RDAPI-30406] Adding note to remove duplicates before upgrading API Gateway

### PR DESCRIPTION
**Pull request by dsantosaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

 Adding a note to check/remove duplicates before upgrading from 7.5.x.

## Deploy preview link

http://pr-2746.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_installation/apigw_upgrade/upgrade_steps_extcass/#step-3---export-the-data-from-the-old-installation-and-upgrade-it

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">âœ…</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">ðŸ”¨</span> Latest commit | 2d4b4ae7 |
 |<span aria-hidden="true">ðŸ˜Ž</span> Deploy preview | http://PR-2746.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_extcass.md [hunk 1]
```diff
@@ -180,6 +180,9 @@ When the installation is complete, perform the new installation checks detailed
 {{< alert title="" color="danger">}}**Caution**</br>
 Before running `export`, you must ensure that the old API Gateway processes are running, including all Admin Node Managers, Node Managers, and API Gateway instances. These processes must be running in the old installation to export the API Gateway configuration data.{{< /alert >}}
 
+{{< alert title="" color="danger">}}**Caution**</br>
+Duplicates should be checked/removed before upgrade from 7.5.x.{{< /alert >}}
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_extcass.md:184]: 'duplicates' of what?

**Review by kmercierwhite — APPROVED** [review]
Looks good to me, except I would argue this could be merged to the master doc branch, rather than the nov23 branch.

**Comment by andreamussap** [comment]
> Looks good to me, except I would argue this could be merged to the master doc branch, rather than the nov23 branch.

I've spoken with JSomerville, he said that it's ok to release it together with Nov23 update

**Pull request merged**